docs/DeveloperGuide.md

```diff
@@ -730,13 +798,45 @@ Similar to [<ins>UC22](#use-case-uc22-assign-a-contact-to-one-or-more-weddings)
     * 2a1. System shows an error message prompting the user to put in a valid index.
 
       Use case resumes at step 1.
+  
+* 2b. The contact is already an assigned Vendor.
```

Comment:
A little bit technical, maybe can change to: "The contact is already a vendor"

Reply:
aight